vcpkg.json

```diff
@@ -1,9 +1,13 @@
 {
     "name": "quickturns",
     "version": "0.0.0",
-    "builtin-baseline": "a7b6122f6b6504d16d96117336a0562693579933",
+    "builtin-baseline": "43235cf746cddb4981a8ef4abdb80aafe4d4e0e2",
```

Comment:
It gives me this error with `"a7b6122f6b6504d16d96117336a0562693579933"`

```
[main] Building folder: QuickTurns 
[build] Starting build
[proc] Executing command: /usr/bin/cmake --build /home/tigran/Documents/_current/QuickTurns/build --config Debug --target all --
[build] [1/1   0% :: 0.001] Re-running CMake...
[build] -- Running vcpkg install
[build] Error: No version entry for boost at version 1.81.0#3. This may be fixed by updating vcpkg to the latest master via `git pull`.
[build] Available versions:
[build]     1.78.0
[build]     1.77.0#2
[build]     1.73.0#3
...
[build]     1.61
[build]     1.60
[build] 
[build] See `vcpkg help versioning` for more information.
[build] Error: No version entry for gtest at version 1.13.0. This may be fixed by updating vcpkg to the latest master via `git pull`.
[build] Available versions:
[build]     1.11.0#3
[build]     1.11.0#2
...
[build]     1.8-1
[build]     1.8
[build] 
[build] See `vcpkg help versioning` for more information.
[build] Error: No version entry for libzip at version 1.9.2. This may be fixed by updating vcpkg to the latest master via `git pull`.
[build] Available versions:
[build]     1.8.0
[build]     1.7.3#2
[build]     1.7.3#1
[build]     1.7.3
[build]     1.7.1#1
...
[build]     1.2.0-2
[build]     1.2.0-1
[build]     1.2.0
[build] 
[build] See `vcpkg help versioning` for more information.
[build] Error: No version entry for libzippp at version 6.0-1.9.2. This may be fixed by updating vcpkg to the latest master via `git pull`.
[build] Available versions:
[build]     5.0-1.8.0#1
[build]     5.0-1.8.0
...
[build]     4.0-1.7.3
[build]     3.1-1.6.1
[build]     2019-07-22
[build] 
[build] See `vcpkg help versioning` for more information.
[build] Error: No version entry for qt5-base at version 5.15.8#4. This may be fixed by updating vcpkg to the latest master via `git pull`.
[build] Available versions:
[build]     5.15.3
[build]     5.15.2#20
[build]     5.15.2#19
...
[build]     5.9.2-2
[build]     5.9.2-1
[build]     5.9.2-0
[build] 
[build] See `vcpkg help versioning` for more information.
[build] Error: No version entry for zlib at version 1.2.13. This may be fixed by updating vcpkg to the latest master via `git pull`.
[build] Available versions:
[build]     1.2.11#13
[build]     1.2.11#12
...
[build]     1.2.8
[build] 
[build] See `vcpkg help versioning` for more information.
[build] -- Running vcpkg install - failed
[build] CMake Error at external/vcpkg/scripts/buildsystems/vcpkg.cmake:834 (message):
[build]   vcpkg install failed.  See logs for more information:
[build]   /home/tigran/Documents/_current/QuickTurns/build/vcpkg-manifest-install.log
[build] Call Stack (most recent call first):
[build]   CMakeLists.txt:16 (include)
[build] 
[build] 
[build] -- Configuring incomplete, errors occurred!
[build] See also "/home/tigran/Documents/_current/QuickTurns/build/CMakeFiles/CMakeOutput.log".
[build] FAILED: build.ninja 
[build] /usr/bin/cmake --regenerate-during-build -S/home/tigran/Documents/_current/QuickTurns -B/home/tigran/Documents/_current/QuickTurns/build
[build] ninja: error: rebuilding 'build.ninja': subcommand failed
[proc] The command: /usr/bin/cmake --build /home/tigran/Documents/_current/QuickTurns/build --config Debug --target all -- exited with code: 1
[build] Build finished with exit code 1
```

Reply:
The baseline number and submodule revision should be the same for it to work, meaning you need to checkout submodule to the same commit. Or simply both those changes can be reverted. 
We can also disable Windows builds for the time being. Even with that fix it failed on main branch because "no space left" error.

Reply:
It's causing issues locally where I need to update the vcpkg every time the latest commit changes running `git submodule foreach git pull origin master`.

It seems like it works without this line as well `"builtin-baseline": "a7b6122f6b6504d16d96117336a0562693579933"`,. Should we consider removing it?

Reply:
Actually I found a way to freeze the vcpkg submodule version from the .gitmodules. That should work now.
https://github.com/tmargary/QuickTurns/blob/cad49153c29a6b5c1dc67ce8aed179d4eb948c95/.gitmodules#L4